app/api/models.py

```diff
@@ -1,10 +1,9 @@
 """Data models."""
 
-from typing import Literal
-
 from fastapi import Query
 from fastapi.exceptions import HTTPException
 from pydantic import BaseModel, constr, root_validator
+from typing_extensions import Literal
```

Comment:
this may be a good idea (I don't know), but let's handle it in a different PR so we don't confuse ourselves later :)

Reply:
It is in a way related to the README.md file update. While I was adding badges, I updated the test suite to run for Python 3.7-3.10. The suite for 3.7 failed due to a discrepancy in the Literal package import statement.

Reply:
ah, great thinking! That's good to check. But also a big enough thing that we should make a separate issue to discuss what versions we want to support and then be intentional about that. So I'd remove it here and bring it back when we address that.